_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -42,20 +42,22 @@ This rule applies to any set of two or more [HTML or SVG elements][] for which a
 - the elements are in the same [web page (HTML)][]; and
 - the elements are [included in the accessibility tree][included in the accessibility tree]; and
 - the elements have [matching][] [accessible names][accessible name] that are not empty (`""`); and
-- have the same [programmatically determined link context][].
+- the elements have the same [programmatically determined link context][].
 
 **Note:** The test target for this rule is the full set of link elements that share the same [matching][] [accessible name][] and [programmatically determined link context][].
 
 ## Expectation
 
-When followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource).
+For the links in each set of target elements, one of the following is true:
+
+- when followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource); or
+- the purpose of the links is [ambiguous to users in general](https://www.w3.org/TR/WCAG21/#dfn-ambiguous-to-users-in-general). 
```

Comment:
"ambiguous to user in general" refers to a single link, not several.
So we should rather say something like
> For each target set, the destination of links that are not ambiguous to users in general are pairwise different

(i.e. remove the ambiguous links from the set and then check that the rest is different)

But then I'm not sure "ambiguous to users in general" get un-ambiguous enough:
> the purpose cannot be determined from the link and all information of the Web page presented to the user simultaneously with the link (i.e., readers without disabilities would not know what a link would do until they activated it)

(see also #1622 on a related ambiguity)

So, maybe wrap it up the other way around:
> For each pair of links in each target set, either the links go to the same place, or there is no visual information on the page to determine they go to different places.

---
(I do believe we need the expectation to looks at every pair of links, rather than the set together, because there may be a mix of ambiguous links and links going to different places which makes the alternative hard to handle)